web/src/components/molecules/Common/ResizableProTable/resizble.tsx

```diff
@@ -0,0 +1,36 @@
+import { Resizable } from "react-resizable";
+import type { ResizeCallbackData } from "react-resizable";
+
+export type { ResizeCallbackData } from "react-resizable";
+
+export const ResizableTitle = (
```

Comment:
This file's name is wrong.
resizble.tsx -> resizable.tsx

Reply:
@KaWaite , fixed.